sampleExtension/optionsPage/htmlFunctionality.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * This file contains the specific functionality for the options and its elements
+ * This file holds all javascript functions used by the html_elements like buttons and tables.
+ * Here things like, onClicked or onChanged events are monitored
+ * The last method: connectHtmlFunctionality is called from the options.js file upon initialization.
+ */
+
+//Local variables that hold the html elements
+var html_table = $('#blacklistTable');
+var html_txtFld = $('#textFld');
+var html_intCnt = $('#iCounter');
+var html_saveButton = $('#saveBtn');
+var html_deleteButton =$('#deleteBtn');
+var modeGroup = "modeOptions";
+
+
+/* -------------------- General functions concerning html_objects -------------------- */
+
+appendHtmlItemTo = function(html_child, html_parent) {
```

Comment:
This is mostly a matter of preference i think, if it turns out to be a bad pattern. I will fix this